File: docs/user/en/get-started/example/connect-demo-server.java
var evita = new EvitaClient(
	EvitaClientConfiguration.builder()
		.host("demo.evitadb.io")
		.port(5556)
        // demo server provides Let's encrypt trusted certificate
	.useGeneratedCertificate(false)
        // the client will not be mutually verified by the server side
        .mtlsEnabled(false)
		.build()
);

File: documentation/user/en/get-started/example/connect-demo-server.cs
var evita = await EvitaClient.Create(
	EvitaClientConfiguration.Builder()
		.Host("demo.evitadb.io")
		.Port(5555)
        // demo server provides Let's encrypt trusted certificate
		.UseGeneratedCertificate(false)
        // the client will not be mutually verified by the server side
        .MtlsEnabled(false)
		.Build()
);
